crypto/ec/ec_lib.c

```diff
@@ -1734,3 +1734,13 @@ EC_GROUP *EC_GROUP_new_from_params(const OSSL_PARAM params[],
 
     return group;
 }
+
+EVP_PKEY *OSSL_EC_keygen(OSSL_LIB_CTX *libctx, const char *curve)
+{
+    OSSL_PARAM params[] =
+        { OSSL_PARAM_construct_utf8_string(OSSL_PKEY_PARAM_GROUP_NAME,
+                                           (char *)curve, 0),
+          OSSL_PARAM_END };
```

Comment:
I was surprised to see this come through the CIs...  but it seems that we have lost the build that checked if the source can be built with `-ansi`, that explains it.  Unfortunately, this construct won't compile...  (I've been hit more than once)

Reply:
This construct is used already several time in crypto and test, apparently without problems.